WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/BloggingPromptsOnboardingUiState.kt

```diff
@@ -10,6 +10,12 @@ sealed class BloggingPromptsOnboardingUiState {
         @StringRes val contentTopRes: Int,
         @StringRes val contentBottomRes: Int,
         @StringRes val contentNoteTitle: Int,
-        @StringRes val contentNoteContent: Int
+        @StringRes val contentNoteContent: Int,
+        @StringRes val primaryButtonLabel: Int,
+        val isPrimaryButtonVisible: Boolean,
+        val onPrimaryButtonClick: () -> Unit,
+        @StringRes val secondaryButtonLabel: Int,
```

Comment:
Added the listeners to the Ready state, so we can test them with real instance of `BloggingPromptsOnboardingUiStateMapper`